### ISSUE TITLE
Delete unused functions from `upstream/account.py`

### ISSUE THREAD

**Issue by scottbarnes** [internetarchive/openlibrary#10223]
### Proposal

We may be able to remove some unused functions from `openlibrary/plugins/upstream/account.py`.

Specifically ones such as:
- `account_password_forgot`
- `account_password_reset`
- `account_email_verify_old`

We should consider removing anything that is no longer in use. The above three may be such examples, and there could easily be more.

#### Requirements Checklist

* [ ]

#### Related files

*

#### Stakeholders

*

<hr>

#### Instructions for Contributors

Please [run these commands](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#working-on-your-branch) to ensure your repository is up to date **before** [creating a new branch](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#making-changes-and-creating-a-pull-request) to work on this issue and **each time after** pushing code to Github, because the pre-commit bot may add commits to your PRs upstream.

**Comment by Spaarsh** [comment]
Can I work on this issue? I will ensure that no tests fails even after deletion of these functions. I had recently worked on removing some unused code from our conftest.py file as well.

**Comment by scottbarnes** [comment]
Thanks, @Spaarsh. I've assigned this to you. Please remove any associated template code, etc., for endpoints that aren't used anywhere.

For endpoints you delete, please also detail where you looked to ensure it's not used (e.g. mention the files and pages you checked to ensure none of them are in use). Sadly, the tests passing may not be a great indicator that the code is unused.

**Comment by Spaarsh** [comment]
@scottbarnes certainly. I also trigger the functions manually and as an additional measure, read git blames to understand why a function was created and why it became redundanct.
